scripts/cmake/Modules/LibAddPlugin.cmake

```
@@ -637,6 +637,8 @@ function (add_plugin PLUGIN_SHORT_NAME)
 
 	if (BUILD_SHARED)
 		add_library (${PLUGIN_NAME} MODULE ${ARG_SOURCES} ${ARG_OBJECT_SOURCES})
+		# TODO: switch to -plugin- in CMake targets
+		set_target_properties (${PLUGIN_NAME} PROPERTIES OUTPUT_NAME "plugin-${PLUGIN_NAME}")
```

```suggestion
		set_target_properties (${PLUGIN_NAME} PROPERTIES OUTPUT_NAME "plugin-${PLUGIN_SHORT_NAME}")
```

See line 430 above. `PLUGIN_NAME` already includes `elektra-`.

---

Thank you!!!!!!!